<launch>
    <!-- mdriver node -->
    <node pkg="mdriver" exec="mdriver_node" name="mdriver_node" namespace="mdriver"/>

    <!-- Node to interface with DAQ card (pickup node) -->
    <node name="pickup_node" pkg="em_tracking" exec="pickup_node">
        <param name="sample_rate" value="200000" />
        <param name="samples_per_channel" value="2000"/>
    </node>

    <!-- Node which enables mdriver and obtains some test measurements -->
    <node pkg="em_tracking" exec="signal_test.py" name="signal_test_node" namespace="signal_test"/>

</launch>